build.gradle

```diff
@@ -15,12 +15,13 @@
  */
 
 plugins {
-    id 'org.ajoberstar.git-publish' version '2.0.0-rc.2' apply false
-    id 'org.curioswitch.gradle-curiostack-plugin'
+    id 'org.ajoberstar.git-publish' version '3.0.0-rc.1' apply false
+    id 'org.curioswitch.gradle-grpc-api-plugin'
```

Comment:
You shouldn't need gRPC (unused) or conda (automatically applied) plugins in the root project